GitCommands/Git/GitItemStatus.cs

```diff
@@ -182,6 +182,25 @@ private void SetFlag(bool isSet, Flags flag)
 
         #endregion
 
+        #region Derived Flags
+
+        /// <summary>
+        /// True if the Git item is added in the (artificial or real) commit.
+        /// </summary>
+        public bool IsAdded
+            => IsNew || IsCopied;
+
+        /// <summary>
+        /// True if Staged is set and the commit is artificial.
```

Comment:
```suggestion
        ///  Indicates whether <see cref="Staged"/> was set and the commit is artificial.
```

I don't believe "artificial" is part of the standard git lingo. Can we use something more git-ish? Like "commit index", "working tree", etc...
Maybe something like this?
```suggestion
        ///  Indicates whether the Git item is yet to be committed; that is it belongs to either... or....
```

Reply:
Artificial is ge specific, but it is more than worktree and index. In this case worktree and index is more correct though, but longer and you get less focus on the functionality.